transport/mqtt/driver.go

```diff
@@ -343,9 +379,35 @@ func (c *MQTTClient) onConnect() {
 
 func (c *MQTTClient) onConnectionLost(err error) {
 	logrus.Warn("Lost connection to MQTT broker")
+
+	// if cert is revoked disconnect from broker and save the error
+	if strings.Contains(err.Error(), "revoked certificate") {
+		logrus.Warn("Disconnecting transport", err.Error())
+		c.lastSeenError = &helpers.DriverError{
+			Err:       helpers.RevokedCertError{Msg: err.Error()},
+			TimeStamp: time.Now(),
+		}
+		c.Disconnect()
+
+	}
 	c.connected = false
 }
 
+func (c *MQTTClient) onReconnecting() {
+
+	logrus.Info("start get limiter token", time.Now())
```

Comment:
Can we just take the time here and only log one line at at the end with `time.Now().Sub(start)`
That we there is only one log line for every reconnect.

Reply:
done!